train.py

```diff
@@ -38,6 +40,7 @@ def train(model_name, seed, epochs, patience, b_size, l_rate, l_decay, minimum_l
     # the data preprocessing per model
 
     dataset = get_dataset(dataset_name)
+    optimizer_hparams = {"lr": l_rate, "weight_decay": l_decay}
```

Comment:
Either this should say "learning_rate_decay" or the parameter should be called "w_decay"

Reply:
This was copied from Rahel's code but I indeed changed the parameter to w_decay